content/en/user-guide/aws/elastic-kubernetes-service/index.md

```diff
@@ -261,14 +269,14 @@ $ curl http://localhost:8081/test123
 {{< / command >}}
 
 {{< alert title="Note" >}}
-You can customize the load balancer port by configuring `EKS_LOADBALANCER_PORT` in your environment.
+You can customize the Load Balancer port by configuring `EKS_LOADBALANCER_PORT` in your environment.
 {{< /alert >}}
 
 ### Enabling HTTPS with local SSL/TLS certificate for the Ingress
 
-In order to enable HTTPS for your endpoints, we can instruct Kubernetes to use SSL/TLS with our [certificate for local domain names](https://github.com/localstack/localstack-artifacts/blob/master/local-certs/server.key) `*.localhost.localstack.cloud`.
+To enable HTTPS for your endpoints, you can configure Kubernetes to use SSL/TLS with your [certificate for local domain names](https://github.com/localstack/localstack-artifacts/blob/master/local-certs/server.key) `*.localhost.localstack.cloud`.
```

Comment:
"your certificate" doesn't seem fully accurate in this context. I'd either leave it as "our certificate" (as we're providing it), or simply use "the certificate".